samples/openapi3/client/petstore/go/go-petstore/model_category.go

```diff
@@ -103,6 +103,12 @@ func (o *Category) SetName(v string) {
 	o.Name = v
 }
 
+// GetDefaultname function assigns the default value &quot;default-name&quot; to the Name field
+// of the Category struct and returns the "default-name".
+func (o *Category) GetDefaultName() interface{}  {
```

Comment:
FYI. filed https://github.com/OpenAPITools/openapi-generator/pull/19268 to update the docstring and improve the code format a bit

Reply:
merged